shared/AppInsightsCommon/src/Interfaces/Context/IApplication.ts

```diff
@@ -11,4 +11,4 @@ export interface IApplication {
      * The application build version.
      */
     build: string;
-}
\ No newline at end of file
+}
```

Comment:
remove the minor code changes that are adding a newline to the end of the file from this PR.